http://bugs.gentoo.org/281690
https://sourceforge.net/tracker/index.php?func=detail&aid=2881185&group_id=53067&atid=469577

the code before AC_PROG_CC isnt terribly important, so ignore it for now so
that we work with autoconf-2.64+

--- libpcap/aclocal.m4
+++ libpcap/aclocal.m4
@@ -44,7 +44,7 @@
 dnl
 AC_DEFUN(AC_LBL_C_INIT,
     [AC_PREREQ(2.12)
-    AC_BEFORE([$0], [AC_PROG_CC])
+    AC_REQUIRE([AC_PROG_CC])
     AC_BEFORE([$0], [AC_LBL_FIXINCLUDES])
     AC_BEFORE([$0], [AC_LBL_DEVEL])
     AC_ARG_WITH(gcc, [  --without-gcc           don't use gcc])
